packages/@aws-cdk/aws-stepfunctions-tasks/lib/index.ts

```diff
@@ -39,8 +39,10 @@ export * from './dynamodb/update-item';
 export * from './dynamodb/delete-item';
 export * from './dynamodb/shared-types';
 export * from './codebuild/start-build';
+export * from './eks/call';
 export * from './athena/start-query-execution';
 export * from './athena/stop-query-execution';
 export * from './athena/get-query-execution';
 export * from './athena/get-query-results';
 export * from './databrew/start-job-run';
+export * from './eks/call';
```

Comment:
duplicated twice... remove one of them please

Reply:
Good catch, will remove in next commit.